src/containers/UserList/UserContextMenuActions.js

```diff
@@ -117,6 +129,20 @@ userContextMenuActions.replicate.subscribe(({ data: user }) => {
     store.dispatch(showDialog(content, props))
 })
 
+userContextMenuActions.reset_password.subscribe(({ data }) => {
+    const snackbarProps = {
+        message: i18n.t(
+            `Are you sure you want to reset {{userName}}'s password?`,
```

Comment:
`i18n` HTML escapes variables by default to prevent XSS vulns, but since React does too this results in double escaping:

![image](https://user-images.githubusercontent.com/4295266/131369420-297f5cca-68b3-4997-8406-1a70ef25d997.png)


```suggestion
            `Are you sure you want to reset {{- userName}}'s password?`,
```

Reply:
Ah yes, good catch. Added it 👍